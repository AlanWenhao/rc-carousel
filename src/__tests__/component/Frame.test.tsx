import { configure, render } from 'enzyme'
import * as Adapter from 'enzyme-adapter-react-16'
import toJson from 'enzyme-to-json'
import 'jest-styled-components'
import * as React from 'react'
import { Frame } from '../../component/Frame'

configure({ adapter: new Adapter() })

describe('Frame', () => {
  it('it works', () => {
    const wrapper = render(<Frame height={200} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
