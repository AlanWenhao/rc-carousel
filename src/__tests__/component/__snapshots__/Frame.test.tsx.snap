// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Frame it works 1`] = `
.c0 {
  display: block;
  width: 100vw;
  height: 200px;
  overflow: hidden;
  background: grey;
  position: relative;
}

<div
  class="c0"
  height="200"
/>
`;
